contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -56,17 +57,19 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
     /**
      * @notice ERC-7201 storage namespace for the vault
      * @dev ERC-7201 namespace is used to prevent upgrade collisions
-     * @custom:report Latest report containing valuation and inOutDelta
-     * @custom:locked Amount of ether locked on StakingVault by VaultHub and cannot be withdrawn by owner
-     * @custom:inOutDelta Net difference between ether funded and withdrawn from StakingVault
-     * @custom:nodeOperator Address of the node operator
-     * @custom:beaconChainDepositsPaused Whether beacon deposits are paused by the vault owner
+     * @custom: report Latest report containing valuation and inOutDelta
```

Comment:
i think the space breaks the natspec custom tag directive

https://docs.soliditylang.org/en/latest/natspec-format.html#tags